### ISSUE TITLE
comico_jp private chapters

### ISSUE THREAD

**Issue by 1271** [yuru-yuri/Manga-Downloader#19]
Some chapters are not downloaded. Example:
http://www.comico.jp/detail.nhn?titleNo=8166&articleNo=107

**Comment by phanirithvij** [comment]
They need some sort of coins which have official payment options and subscriptions 

Only the chapters that have __"free"__ in japanese __"無料"__ on the right side
can be downloaded
I don't think we can download all the chapters

### Screen Shot (google translated the 'free' word)

![comico](https://user-images.githubusercontent.com/29627898/41775285-fd06fba0-7640-11e8-9e86-fdd7be17ac8d.png)


### Screen shot (the japanese(original) one)

![comicojap](https://user-images.githubusercontent.com/29627898/41775375-4ba9a442-7641-11e8-8ec8-cbc2cd199e02.png)

**Comment by phanirithvij** [comment]
They are directly sending image urls from the server side and not by using API.

So there is no way as of now to get all the chapters ( private )

**Comment by 1271** [comment]
Yes I know. Therefore, the option of entering a login-password was initially considered. This will be done once, I think.

**Comment by z8512** [comment]
Using --arguement flag we can login right?
